.github/ISSUE_TEMPLATE/archived.md

```diff
@@ -3,19 +3,19 @@ name: Archiving checklist
 about: archiving
 title: "[ARCHIVE] project"
 labels: archive
-assignees: caniszczyk, amye, idvoretskyi, jeefy
+assignees: caniszczyk, mrbobbytables, idvoretskyi, jeefy, castrojo
 ---
 - [ ] TOC vote to archive passed
-- [ ] Move logo to archived on CNCF.io	
-- [ ] Remove from landscape		
-- [ ] Archive maintainers mailing list	
-- [ ] Remove ML from maintainers@ mailing group
-- [ ] Remove/Archive on maintainers.cncf.io
-- [ ] Remove logo from all slide decks	
-- [ ] Remove logo from event sites	
-- [ ] Move logo to archived in artwork	
-- [ ] Archive DevStats (or archive?)	
-- [ ] Notify Speakers team to remove from CFPs	
-- [ ] Remove from service desk or archive	
-- [ ] Remove from CNCF store
+- [ ] Move project (logo) to archived in the [artwork repo](https://github.com/cncf/artwork)
+- [ ] Move project to archived on [landscape](https://github.com/cncf/landscape)
+- [ ] Remove maintainer list from [project-maintainers.csv](https://github.com/cncf/foundation/blob/main/project-maintainers.csv)
+- [ ] Archive (lock) maintainers mailing list (groups.io)
+- [ ] Remove ML from maintainers@ mailing group (google groups)
```

Comment:
Nit: what is ML short for?

Reply:
mailing list, the maintainers@ list does not have individuals but the maintainer mailing lists of every project:👍